src/Modules/clearCompleted.js

```diff
@@ -0,0 +1,23 @@
+import { updatingstorage } from './task.js';
+import sortOutput from './sort.js';
+
+const clearButton = () => {
+  const removeCompleted = document.querySelector('.reset');
+  removeCompleted.addEventListener('click', () => {
+    const taskstatus = document.querySelectorAll('.taskstatus');
+    taskstatus.forEach((task) => {
+      if (task.checked) {
+        task.parentNode.remove();
+      }
+    });
+
+    const afterClear = sortOutput();
+    const filtered = afterClear.filter((elem) => elem.completed !== true);
+    for (let i = 0; i < filtered.length; i += 1) {
+      filtered[i].index = `${i}`;
+    }
```

Comment:
You have good and clean code here but in this section i believe it is better to use a foreach instead of for loops 
Check the link at number 3 
- [ https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec]